tests/e2e/default/tasks/tasks.wdio-spec.js

```diff
@@ -55,7 +55,8 @@ describe('Tasks', () => {
     await utils.revertSettings(true);
   });
 
-  it('Should show error message for bad config', async () => {
+  // Skipped due to error: done() called multiple times
```

Comment:
Same feedback about comments, this seems like a ToDo to fix now, or it should link a ticket to fix it next. 

```
// ToDo: Fix XXXX. Ref: https://github.com/medic/[issue_number]
```